packages/vaadin-combo-box/theme/lumo/vaadin-combo-box-item-styles.js

```diff
@@ -1,47 +1,48 @@
 import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { item } from '@vaadin/vaadin-item/theme/lumo/vaadin-item-styles.js';
```

Comment:
We can remove `vaadin-item.js` import below now when we import its styles explicitly.
Note that you already did it for Material theme, so let's change it here as well.

Reply:
Done.